contracts/Vault.vy

```diff
@@ -444,7 +444,7 @@ def setLockedProfitDegration(degration: uint256):
     @param degration The rate of degration in percent per second scaled to 1e18.
     """
     assert msg.sender == self.governance
-    assert degration >= 0 and degration <= DEGREDATION_COEFFICIENT
+    assert degration <= DEGREDATION_COEFFICIENT
```

Comment:
Due to the way the this is structured it's not possible to obtain full coverage for this line. Since "degration" is a uint256 it can never be less than zero (you will get an outside of range error if you try). The only way to obtain full coverage here is to change the assert statement in Vault.vy.

Reply:
Add a comment above this